app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% flash.each do |type, msg| %>
+  <div>
+    <font size="5" color="red"><%= msg %></font>
+  </div>
```

Comment:
Flash message gets rendered here. Might not need this, look the code over and change it if possible.